docs/usage/configuration-options.md

```diff
@@ -1384,6 +1384,10 @@ The description field can be used inside any configuration object to add a human
 A description field embedded within a preset is also collated as part of the onboarding description unless the preset only consists of presets itself.
 Presets which consist only of other presets have their own description omitted from the onboarding description because they will be fully described by the preset descriptions within.
 
+## detectBaseBranchConfigFileName
```

Comment:
move this feature to a separate PR

Reply:
I'm happy for this to be default behavior actually

Reply:
sure, but its unrelated to the feature this PR is trying to implement. that's why I suggested to move it to a separate PR.

Reply:
Good point, we should have "detection of a different file name in base branch" before or after this